### ISSUE TITLE
Ability to Combine Two or more fields into One Field

### ISSUE THREAD

**Issue by EquilibriumTechx** [CrocoBlock/suggestions#5486]
HI, 

I would like to suggest this, to add a jet engine capability to combine 1 or more fields into one field with a separator of choice, for example " , / | or & + - " and to display them on another field example 

text Field 1 = 12345
text field 2 = 90866
Seperator of choice = ,
text field 3 = 12345,90866

This is really helpful

**Comment by machachen** [comment]
Agree. It would be a really powerful. I now just do the calculation outside and add an extra custom field to the post type.

**Comment by Lonsdale201** [comment]
https://crocoblock.com/knowledge-base/articles/jetengine-devtools-calculated-callback-addon/

**Comment by EquilibriumTechx** [comment]
> https://crocoblock.com/knowledge-base/articles/jetengine-devtools-calculated-callback-addon/

Yes this works for front end and it's to calculate and stuff, I'm referring to back end jet engine section and just to combine fields with a separator, not to calculate

**Comment by filwi** [comment]
A simple string concactenator would be very useful!

**Comment by dmayembe** [comment]
Yes, a combination of strings. not just calculated filed with arithmetic

**Comment by bjoernzosel** [comment]
+1

**Comment by olyaastep** [comment]
+1

**Comment by eaglevu** [comment]
This and/or key generator, but string concatenation based on fields will get us a long way.

**Comment by Salimoh** [comment]
+1

**Comment by spacehoboguy** [comment]
+1

**Comment by abdullahwp** [comment]
you can achieve this by doing this "text-field" + " " + "text-field-2" in calculated field 

 "first-name" + " " + "last-name"

i do this to save both in 1 meta field

also make it hidden

**Comment by RicoGL** [comment]
> you can achieve this by doing this "text-field" + " " + "text-field-2" in calculated field
> 
> "first-name" + " " + "last-name"
> 
> i do this to save both in 1 meta field
> 
> also make it hidden

How does it work? Whatever I try I get "undefined" in the calculated field.
